src/passes/Print.cpp

```diff
@@ -1415,6 +1416,16 @@ struct PrintSExpression : public OverriddenVisitor<PrintSExpression> {
       if (iter != debugLocations.end()) {
         printDebugLocation(iter->second);
       }
+      // show a binary position, if there is one
+      if (debugInfo) {
+        auto iter = currFunction->binaryLocations.find(curr);
```

Comment:
so concretely could this be something like `o << "code offset " << curr->binaryLocation`

Reply:
That's a good point; although since the offset is just a 32-bit number, and the cost of a map entry is at least the pointer-sized key plus the value (plus any overhead; bucket chain?), there would have to be info for fewer than 1/3 of the nodes for that to be an improvement.